### ISSUE TITLE
BorgBS :)

### ISSUE THREAD

**Issue by ThomasWaldmann** [marcpope/bbs#1]
Had to laugh when reading it abbreviated that way.

**Comment by marcpope** [comment]
Yeah well BBS's aren't really around much any more! :)  Fun fact: I used to run a FidoNet back in the 80s

**Comment by ThomasWaldmann** [comment]
I ran a fido hub system back then. OS/2 powered. :D

**Comment by poelzi** [comment]
@ThomasWaldmann every good node was os/2 powered ;) I was just a fidonet user - ahh memories.... If you had your node around Stuttgart I may have connected once ...

**Comment by marcpope** [comment]
I also ran a BBS for a while, it was called "The Tavern" it ran on an Apple //e with 3 Apple Cat modems. Those were blazin at 1200 baud half duplex!
